env/staging/cloudwatch_alarms/terragrunt.hcl

```diff
@@ -4,6 +4,7 @@ inputs = {
   sns_topic_critical_name  = "alert-critical"
 
   feature_api_alarms              = true
+  api_gateway_400_error_threshold = 95
```

Comment:
This might be too high, although we do still have a large number of phones reporting to staging.

Reply:
I've left this as-is for now and we can tweak if needed.